internal/updater/updater_test.go

```diff
@@ -0,0 +1,256 @@
+package updater_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	"github.com/operator-framework/rukpak/internal/updater"
+)
+
+var _ = Describe("Updater", func() {
+	var (
+		client client.Client
+		u      updater.Updater
+		obj    *rukpakv1alpha1.Bundle
+		status = &rukpakv1alpha1.BundleStatus{
+			Info:               &rukpakv1alpha1.BundleInfo{},
+			Phase:              rukpakv1alpha1.PhaseFailing,
+			Digest:             "digest",
+			ObservedGeneration: 1,
+			Conditions: []metav1.Condition{
+				{
+					Type:               "Working",
+					Status:             metav1.ConditionTrue,
+					ObservedGeneration: 3,
+					LastTransitionTime: metav1.Time{},
+					Reason:             "requested",
+					Message:            "Working correctly",
+				},
+				{
+					Type:               "starting",
+					Status:             metav1.ConditionTrue,
+					ObservedGeneration: 1,
+					LastTransitionTime: metav1.Time{},
+					Reason:             "started",
+					Message:            "starting up",
+				},
+			},
+		}
+	)
+
+	BeforeEach(func() {
+		schemeBuilder := runtime.NewSchemeBuilder(
+			kscheme.AddToScheme,
+			rukpakv1alpha1.AddToScheme,
+		)
+		scheme := runtime.NewScheme()
+		Expect(schemeBuilder.AddToScheme(scheme)).ShouldNot(HaveOccurred())
+
+		client = fake.NewClientBuilder().WithScheme(scheme).Build()
+		u = updater.New(client)
+		obj = &rukpakv1alpha1.Bundle{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "testBundle",
+				Namespace: "testNamespace",
+			},
+			Spec: rukpakv1alpha1.BundleSpec{
+				ProvisionerClassName: "core.rukpak.io/plain-v0",
```

Comment:
This spec.provisionerClassName was very recently updated in #144 to simply `plain` -- I know you have changed this once in the PR already, sorry about this, but could you update it again? After that this is good to merge. 

It looks like this doesn't affect the test results, since a fake client is being used and no reconciliation is occurring, but it would still be nice to have a consistent naming convention in the project.

Reply:
I think this could also be a package-level constant to improve readability + maintainability over time (i.e I only need to make a provisioner class name change at one level vs. throughout multiple spots within a package).